api/app/routes/feedback.py

```
@@ -100,3 +100,70 @@ async def get_feedback_stats():
             status_code=500,
             detail="An error occurred while retrieving feedback statistics"
         )
+
+
+@router.post("/feedback/explanation", response_model=Dict[str, Any])
+async def submit_feedback_explanation(
+        request: Request,
+        explanation_data: Dict[str, Any] = Body(...),
```

Ruff (0.8.2)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)